ArduPlane/Parameters.cpp

```diff
@@ -650,12 +644,6 @@ const AP_Param::Info Plane::var_info[] = {
     // @User: Advanced
     GSCALAR(reset_switch_chan,      "RST_SWITCH_CH",  0),
```

Comment:
why not this one too?

Reply:
because I did not find it, ;) I will add it too.

Reply:
done